pkg/cache/v2/cache.go

```diff
@@ -69,7 +69,6 @@ type Response interface {
 // RawResponse is a pre-serialized xDS response containing the raw resources to
 // be included in the final Discovery Response.
 type RawResponse struct {
-	Response
```

Comment:
I don't quite understand the reason for embedding here... Any insight?

Reply:
@jessicayuen Is it OK to remove this? There is no explanation why this field is defined and it builds fine without it.

Reply:
I believe the intention here was to make it mandatory for `RawResponse` and `PassthroughResponse` to implement the contract in the `Response` interface.  Otherwise, accidental removal of one of the [methods](https://github.com/envoyproxy/go-control-plane/pull/326/files#diff-559e68cd870b75c9dbc9c282d7592466L143) or changes to the `Response` interface would make these structs incompatible with the `CreateWatch` and `Cache` functionalities.

Reply:
I see. I think the idiomatic way to do that is with `var _ Response = &RawResponse`. Adding an anonymous field would just delegate the calls to the inner Response field, see https://putridparrot.com/blog/anonymous-fields-in-go/ for example.

Reply:
I see, ok thanks for the fix!